Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2434 @@
+
+    row = summary_data[0] if summary_data else {}
+    headers = list(row.keys())
+    column_order = list(camelize_string(column) for column in headers)
+    # , headers=headers,
+    markdown = tableToMarkdown(title, camelize(summary_data, '_'), headers=column_order, removeNull=True)
+    return markdown, context_data
+
+
+def endpoint_instance_readable_output(results: list[dict], title: str) -> str:
+    """
+    Convert to XSOAR Readable output for entities endpoints instance
+    Args:
+        results (list): Symantec Association Results data
+        title (str): Title string
+    Returns:
+        A string representation of the Markdown table
+    """
+
+    summary_data = []
+    for data in results:
+        ip_addresses = data.get("ip_addresses", [])
+        new = {
+            'device_uid': data.get('device_uid', ''),
+            'device_name': data.get('device_name', ''),
+            'device_ip': data.get('device_ip', ''),
+            'domain_or_workgroup': data.get('domain_or_workgroup', ''),
+            'time': data.get('time', ''),
+            'ip_addresses': ip_addresses
+        }
+        summary_data.append(new)
+
+    headers = summary_data[0] if summary_data else {}
+    headers = list(headers.keys())
+    column_order = list(camelize_string(column) for column in headers)
+    markdown = tableToMarkdown(title, camelize(summary_data, "_"), headers=column_order,
+                               removeNull=True)
+    return markdown
+
+
+def incident_readable_output(results: list[dict], title: Optional[str] = None):
+    """
+    Convert to User Readable output for Incident resources
+    Args:
+        results (list): Symantec Association Results data
+        title (str): Title string
+    Returns:
+        markdown: A string representation of the Markdown table
+        summary_data : Formatting response data
+    """
+    summary_data: list[dict[str, Any]] = []
+    for data in results:
+        priority = data.get('priority_level', '')
+        state = data.get('state', '')
+        resolution = data.get('resolution', '')
+        new = {
+            # EDR CONSOLE Headers : ID , Description, incident Created, Detection Type, Last Updated,priority
+            'incident_id': data.get('atp_incident_id', ''),
+            'description': data.get('summary', ''),
+            'incident_created': data.get('device_time', ''),
+            'detection_type': data.get('detection_type', ''),
+            'last_updated': data.get('updated', ''),
+            'priority': INCIDENT_PRIORITY_LEVEL.get(str(priority), ''),
+            # ------------------
+            'incident_state': INCIDENT_STATUS.get(str(state), ''),
+            'atp_rule_id': data.get('atp_rule_id'),
+            'rule_name': data.get('rule_name'),
+            'incident_uuid': data.get('uuid'),
+            'log_name': data.get('log_name'),
+            'recommended_action': data.get('recommended_action'),
+            # 'summary': data.get('summary'),
```

Comment:
is this commented out on purpose?

Reply:
Yes, commented to remove it duplicate with 'description': data.get('summary', ''),

Reply:
Not exists now.